docs/reference/standard-library/std/banker.md

```diff
@@ -0,0 +1,96 @@
+---
+id: banker
+---
+
+# Banker
+View concept page [here](../../../concepts/standard-library/banker.md).
+
+```go
+type BankerType uint8
+
+const (
+    BankerTypeReadonly BankerType = iota
+    BankerTypeOrigSend
+    BankerTypeRealmSend
+    BankerTypeRealmIssue
+)
+
+type Banker interface {
+    GetCoins(addr Address) (dst Coins)
+    SendCoins(from, to Address, amt Coins)
+    IssueCoin(addr Address, denom string, amount int64)
+    RemoveCoin(addr Address, denom string, amount int64)
+}
+```
+
+## GetBanker
+Returns `Banker` of the specified type.
+
+#### Parameters
+- `BankerType` - type of Banker to get:
+    - `BankerTypeReadOnly` - read-only access to coin balances
+    - `BankerTypeOrigSend` - full access to coins sent with the transaction that calls the banker
+    - `BankerTypeRealmSend` - full access to coins that the realm itself owns, including the ones sent with the transaciton
+    - `BankerTypeRealmIssue` - able to issue new coins
+
+#### Usage
+
+```go
+banker := std.GetBanker(std.<BankerType>)
+```
+---
+
+## GetCoins
+Returns `Coins` owned by `Address`.
+
+#### Parameters
+- `addr` **Address** to fetch balances for
+
+#### Usage
+
+```go
+coins := banker.GetCoins(addr)
+```
+---
+
+## SendCoins
+Sends `amt` from address `from` to address `to`. `amt` needs to be a well-defined
```

Comment:
"`amt` needs to be a well-defined `Coins` slice.." then we see below that `amt` is of type `int64` rather than `[]Coin`

Reply:
This is a naming inconsistency in the functions. I see what you're saying - it can be confusing. For the SendCoins function, it is indeed a `Coins` struct, while in RemoveCoin it is just a single `Coin`. Let's refactor in VM?

Reply:
Fixed: [3012f21](https://github.com/gnolang/gno/pull/1468/commits/3012f21b1f833de06fe6e247608464477e14d7c7)